### ISSUE TITLE
Make the UI WCAG2 compatible - Part 2

### ISSUE THREAD

**Issue by pawelczak** [CeON/saos#450]
- analyse color contrast,
- check links with Accessibility Extension,
- wc3 validator,
- page title,
- check 200% page size,
- change elements with background-image to img (...),
- text on site should be displayed propely, after css styles are disabled
